src/components/NewRuntimeModal.js

```diff
@@ -453,6 +449,22 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
     return _.find(({ imageType }) => _.includes(imageType, ['Jupyter', 'RStudio']), runtimeDetails?.runtimeImages)?.imageUrl
   }
 
+  getCurrentMountDirectory(currentRuntimeDetails) {
+    if (currentRuntimeDetails?.labels.tool) {
+      return currentRuntimeDetails?.labels.tool === 'RStudio' ? '/home/rstudio' : '/home/jupyter-user/notebooks'
+    } else {
+      return ''
```

Comment:
In js I think the convention is using "null" or "undefined" in a case like this rather than ''

Reply:
This var is displayed in a string, so I think we want it to be empty in the case there isn't a mount Dir so we dont display null or undefined. However, we could do something like `'[No mount dir]'` to be explicit. I think this is fine though.